.github/stale.yml

```diff
@@ -0,0 +1,24 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 30
```

Comment:
Perhaps make this 2 weeks and see how that goes? I've not often seen issues that are not yet triaged and also being resolved after a couple of weeks, perhaps the bot commenting earlier about the staleness might prompt a response on e.g. missing information?